manifest/ocischema/manifest.go

```diff
@@ -0,0 +1,133 @@
+package ocischema
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/docker/distribution"
+	"github.com/docker/distribution/manifest"
+	"github.com/opencontainers/go-digest"
+)
+
+const (
+	// MediaTypeManifest specifies the mediaType for the current version.
+	MediaTypeManifest = "application/vnd.oci.image.manifest.v1+json"
+
+	// MediaTypeConfig specifies the mediaType for the image configuration.
+	MediaTypeConfig = "application/vnd.oci.image.config.v1+json"
+
+	// MediaTypePluginConfig specifies the mediaType for plugin configuration.
+	MediaTypePluginConfig = "application/vnd.docker.plugin.v1+json"
+
+	// MediaTypeLayer is the mediaType used for layers referenced by the manifest.
+	MediaTypeLayer = "application/vnd.oci.image.layer.v1.tar+gzip"
```

Comment:
I think this needs updating for the newest version of the OCI Image spec which has 4 types of layer media types.